src/layers/legacy/medCoreLegacy/medExternalResources.h

```diff
@@ -23,19 +23,20 @@
 
 
 #include <QString>
+#include <medCoreLegacyExport.h>
 
 namespace med
 {
 
 // Returns the directory containing the external resources for the project or
 // one of its libraries. A null string is returned if the directory does not
 // exist.
-QString getExternalResourcesDirectory(QString libraryName = QString());
+    MEDCORELEGACY_EXPORT QString getExternalResourcesDirectory(QString libraryName = QString());
```

Comment:
Throughout medPython I did not indent namespace content (as I follow the google style guide). You indented these lines, but is it a typo or is that the style you follow?

Reply:
Ho sorry it's my style code, but in your code I follow your style.
I will change it.

Reply:
done